.circleci/config.yml

```diff
@@ -282,7 +298,7 @@ commands:
           #
 
   conditionally-launch-build-and-deploy:
-    description: Launch build and deploy if project has changed
+    description: Check every study, launch build and deploy if study has changed
```

Comment:
`conditionally-launch-build-and-deploy` command actually loops thru all studies and determine if build and store is needed for every study. Command name is not clear about this. I don't want to rename it in this PR because it could cause more confusion.